openpype/plugins/publish/extract_review_slate.py

```diff
@@ -314,6 +308,41 @@ def process(self, instance):
                 slate_subprocess_cmd, shell=True, logger=self.log
             )
 
+            # Create slate with silent audio track
+            if input_audio:
```

Comment:
How this second "silent" output is integrated? Because it's created but unused...